### ISSUE TITLE
提几点建议功能

### ISSUE THREAD

**Issue by abc1763613206** [luogu-dev/luogu-blog-frontend#17]
具体在洛谷群里已经说明过了:
- 支持同步滚动（左端与右端）
- 支持类似Github的访问链接直接跳转到指定章节
- 类似参考资料的脚注（联动上一点）

**Comment by moesoha** [comment]
@abc1763613206 同步滚动应当是是编辑器 markdown-palettes 的功能，并且这个功能已经实现了，只是博客这边没有更新。

**Comment by abc1763613206** [comment]
那么部署周期多长呢？

**Comment by memset0** [comment]
`id`也可以通过页面的js代码来设置吧（也许）

**Comment by TitanSnow** [comment]
> id也可以通过页面的js代码来设置吧（也许）

Of course, and that is exactly what I meant. This process should be done in blog frontend, not in the editor component.

**Comment by memset0** [comment]
@TitanSnow Sorry, because of my poor English I cannot understand what you meant at first.

**Comment by abc1763613206** [comment]
所以这个凉了？

**Comment by Bambusaceae** [comment]
> 支持类似Github的访问链接直接跳转到指定章节

这个貌似可以通过编辑器实现？
